docs/release-notes/7.15.asciidoc

```diff
@@ -94,3 +93,8 @@
 * Adds the `Responses` field to telemetry ({pull}111892[#111892]).
 * Fixes issues with the pagination on the Exceptions table ({pull}111000[#111000]).
 * Fixes a bug that caused empty comments to display in an endpoint's activity log ({pull}111163[#111163]).
+
+[discrete]
+[[known-issue-7.15.0]]
+==== Known issues
+* Case comments containing https://github.github.com/gfm/#what-is-github-flavored-markdown-[GitHub Flavored Markdown (GFM)] will cause migrations to fail when upgrading to {stack} version 7.15.0 or later. To prevent this, remove GFM from the case comment before upgrading ({pull}1195093[#119509]).
```

Comment:
I think there's an extra `3` in the issue link `({pull}1195093[#119509]).` When I navigate to it, it takes me to a 404 page. I think `119509` is what we want.

Reply:
Yes, removing that now. It should be pointing to https://github.com/elastic/kibana/issues/119509. Thanks!